pkg/ingestor/parser/spdx/parse_spdx.go

```diff
@@ -60,61 +69,74 @@ func (s *spdxParser) Parse(ctx context.Context, doc *processor.Document) error {
 func (s *spdxParser) getTopLevelPackage() {
 	// oci purl: pkg:oci/debian@sha256%3A244fd47e07d10?repository_url=ghcr.io/debian&tag=bullseye
 	splitImage := strings.Split(s.spdxDoc.DocumentName, "/")
+	var purl string
 	if len(splitImage) == 3 {
-		topPackage := assembler.PackageNode{}
-		topPackage.Purl = "pkg:oci/" + splitImage[2] + "?repository_url=" + splitImage[0] + "/" + splitImage[1]
-		topPackage.Name = s.spdxDoc.DocumentName
-		topPackage.Tags = []string{"container"}
-		topPackage.NodeData = *assembler.NewObjectMetadata(s.doc.SourceInformation)
-		s.packages[string(s.spdxDoc.SPDXIdentifier)] = append(s.packages[string(s.spdxDoc.SPDXIdentifier)], topPackage)
+		purl = "pkg:oci/" + splitImage[2] + "?repository_url=" + splitImage[0] + "/" + splitImage[1]
```

Comment:
I thought we didnt want to do this going forward, as it was very error-prone.

Reply:
Should this also be a `asmhelpers.GuacPkgPurl`?

Reply:
I will add a TODO and handle this during the next PR